packages/vscode-extension/src/handlers.ts

```diff
@@ -1626,7 +1626,10 @@ export async function showError(e: UserError | SystemError) {
       },
     };
 
-    const button = await window.showErrorMessage(`[${errorCode}]: ${e.message}`, help);
+    const button = await window.showErrorMessage(
+      `[${errorCode}]: ${e.userData ?? e.message}`,
```

Comment:
Is there any other code using `userData`? Per the comment of userData, seems it's for other purpose